notebooks/Notebook3.md

```diff
@@ -2,33 +2,39 @@
 
 ## Last week's critique
 
-**TODO:** Fill in this part with a summary and reflection on the critique you received for
-last week's work. Answer questions such as:  How, specifically, did the feedback help
-improve the project? Did the feedback point out or offer something you hadn't considered?
-Did it help you make a design decision? Was it helpful in addressing the most pressing
-issues in your project? How will you incorporate the feedback into your work? Will you
-change something about the design, implementation, or evaluation as a result?
+I definitely needed the help this week. Charlie helped on monday and Prof Ben helped on Wednesday. I think we are all dealing with that classic project situation of biting off more then we could chew. Im happy with the progress im making and the changes I can make, but im defintely far from the original goal. 
```

Comment:
Huge agree here- without all the help I've been getting in class from you and Prof Ben, I would have quit the project long ago; even though I had a really big setback this week I'm still feeling excited to work more on the project.